templates/incoming_ref_missing.j2

```diff
@@ -0,0 +1,36 @@
+Hello,
+we're closing this pull request because the original PR-bound branch
+went missing. (This PR's fork repository or branch has been deleted by
+the owner).
+
+@{{ submitter }}, if you with to restore the PR, please, re-create
+a branch in the same fork repo and with the exactly same name and reopen
+this pull request. Or submit a new one.
+
+##### To whom it may concern,
+If you're an individual wishing to pick up the work done by the original
+submitter (@{{ submitter }}), [here's how you can do this](
+https://help.github.com/articles/checking-out-pull-requests-locally/#modifying-an-inactive-pull-request-locally)
+(given you followed the instructions at [Ansible Dev Guide](
+https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html#configuring-your-remotes)
+and have `origin` pointing to your fork and `upstream` pointing to
+`ansible/ansible`):
+
+1. :ambulance: Pull the branch locally from read-only copy held by GitHub:
```

Comment:
Please remove the emojis, in this cause I don't think they add much https://gist.github.com/gundalow/7d080d8787bc19bb11023713ebd241ec

Reply:
I think they add bits of humanity to this message. I'd like to keep them. Maybe just less of them?